CMake/External_OpenCV.cmake

```diff
@@ -197,6 +222,7 @@ ExternalProject_Add(OpenCV
     -DBUILD_SHARED_LIBS:BOOL=True
     -DBUILD_TESTS:BOOL=False
     -DWITH_EIGEN:BOOL=${fletch_ENABLE_EIGEN}
+    -DWITH_JASPER:BOOL=False
```

Comment:
It is disabled in both cases (this is in the main external_project_add for opencv)